next/src/components/_global/cardsComponent/cardsComponent.tsx

```diff
@@ -0,0 +1,72 @@
+import styles from './cardsComponent.module.scss';
+import { type Cards } from '@/types/_global/Cards';
+import CentralizedHeading from '@/components/ui/centralizedHeading/centralizedHeading';
+import Img from '@/components/ui/Img';
+import Markdown from '@/components/ui/Markdown';
+
+export default function CardsComponent({ data: { centralizedHeading, cards } }: { data: Cards }) {
+  return (
+    <section className={styles.cardsComponent}>
+      <div className={` ${styles.componentWrapper} maxWidth`}>
```

Comment:
Don't add maxWidth to every component. Simply add the `max-width` declaration along with `margin: 0 auto` to the `main` element.

Reply:
That won't work. There are backgrounds on full-width for some sections. The solution that you have provided won't do.

Reply:
This is possible. Check it here: https://github.com/KryptonumDev/kierunek-dzierganie/blob/1254c3e731fed44dfe34f23c7437082b2b9ae7b8/src/global/global.scss#L174

Reply:
This change should be made in dev branch. There will be conflicts otherwise